packages/kbn-optimizer/limits.yml

```diff
@@ -167,7 +167,7 @@ pageLoadAssetSize:
   snapshotRestore: 22068
   spaces: 30665
   stackAlerts: 31499
-  stackConnectors: 80788
+  stackConnectors: 159130
```

Comment:
Is this something we can look into reducing?

Reply:
~~Thanks for reviewing @jbudz, we were importing the zod library but I've changed it to load async. It's on register step, so IIUC it shouldn't affect user UX. It's down to 81160 now~~

Now it's not working, will come back again once I figured out what's goin on